bin/pygrb/pycbc_grb_trig_combiner

```diff
@@ -371,6 +371,13 @@ parser.add_argument(
     default=6,
     help="The number of off source trials, default: %(default)d",
 )
+parser.add_argument(
+    "-e",
+    "--seed",
+    type=int,
+    default=1234,
```

Comment:
Do we want a integer default or `None`? For example, this will always give us the same set of slide IDs for every GRB with a maximal offsource.

Reply:
Good point.  I will change the default `None`.